app/src/main/res/layout/item_comment_video.xml

```diff
@@ -11,35 +11,29 @@
 
     </data>
 
-    <android.support.v7.widget.CardView
+
+    <android.support.constraint.ConstraintLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         >
-        <android.support.constraint.ConstraintLayout
+        <include
+            layout="@layout/item_comment_only_text"
+            android:id="@+id/layout_user"
+            app:commentViewModel="@{commentViewModel}"
+            />
+        <com.google.android.exoplayer2.ui.SimpleExoPlayerView
+            android:id="@+id/video_view"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            >
-            <include
-                layout="@layout/item_comment"
-                android:id="@+id/layout_user"
-                app:commentViewModel="@{commentViewModel}"
-                />
-            <com.google.android.exoplayer2.ui.SimpleExoPlayerView
-                android:id="@+id/video_view"
-                android:layout_width="match_parent"
-                android:layout_height="@dimen/dp_200"
-                android:layout_marginBottom="@dimen/dp_8"
-                android:layout_marginLeft="@dimen/dp_56"
-                android:layout_marginRight="@dimen/dp_8"
-                app:controller_layout_id="@layout/custom_video_playback_controller"
-                app:layout_constraintLeft_toLeftOf="parent"
-                app:layout_constraintRight_toRightOf="parent"
-                app:layout_constraintTop_toBottomOf="@+id/layout_user"
-                app:layout_goneMarginBottom="@dimen/dp_8"
-                app:layout_goneMarginTop="@dimen/dp_8"
-                app:resize_mode="fixed_height"
-                app:use_controller="false"
-                />
-        </android.support.constraint.ConstraintLayout>
-    </android.support.v7.widget.CardView>
+            android:layout_height="@dimen/dp_200"
+            android:layout_marginBottom="@dimen/dp_8"
+            android:layout_marginLeft="@dimen/dp_56"
+            android:layout_marginRight="@dimen/dp_8"
```

Comment:
WARNING: [RtlHardcoded] Consider adding `android:layout_marginEnd="@dimen/dp_8"` to better support right-to-left layouts
 Using `Gravity#LEFT` and `Gravity#RIGHT` can lead to problems when a layout is rendered in locales where text flows from right to left. Use `Gravity#START` and `Gravity#END` instead. Similarly, in XML `gravity` and `layout_gravity` attributes, use `start` rather than `left`.

For XML attributes such as paddingLeft and `layout_marginLeft`, use `paddingStart` and `layout_marginStart`. *NOTE*: If your `minSdkVersion` is less than 17, you should add *both* the older left/right attributes *as well as* the new start/right attributes. On older platforms, where RTL is not supported and the start/right attributes are unknown and therefore ignored, you need the older left/right attributes. There is a separate lint check which catches that type of error.

(Note: For `Gravity#LEFT` and `Gravity#START`, you can use these constants even when targeting older platforms, because the `start` bitmask is a superset of the `left` bitmask. Therefore, you can use `gravity="start"` rather than `gravity="left|start"`.)